tests/e2e/test_pvc_creation_deletion_scale.py

```diff
@@ -0,0 +1,254 @@
+"""
```

Comment:
@ramkiperiy this test file already exists - https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_pvc_creation_deletion_scale.py

Reply:
@ebenahar created this PR just to run testcase over jenkins and the contents are from old branch. 
Wa trying to debug the issue we are observing during scale test case run, this particular TC https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_pvc_creation_deletion_scale.py failed in all 4 runs.
i.e. https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/6300/

Figured out the issue is due to log generations time delay, i.e. after deleting 1500 PVC part of script collecting all the logs immediately so due to this some pvc logs are missing. Planning to add a 2 min wait time before collecting complete logs.

Anyway will be creating a fresh PR to fix the issue and delete this PR.